ui/src/Elements/Message/Message.view.js

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable react/no-multi-comp */ // disabled as we have a hoc funtion in file
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  ClickableTile,
+  ExpandableTile,
+  Tile,
+  TileAboveTheFoldContent,
+  TileBelowTheFoldContent,
+} from 'carbon-components-react';
+import { CheckmarkFilled16, ErrorFilled16 } from '@carbon/icons-react';
+import { isEmpty, isFunction } from 'lodash-es';
+import clsx from 'clsx';
+
+import { Body } from '../Text/index.js';
+import { useTranslate } from '../../ReactCustomHooks/index.js';
```

Comment:
Being hyper picky, could these not be just from `Elements` and `ReactCustomHooks`?

Reply:
Done.